style/style.css

```diff
@@ -412,3 +428,210 @@ form {
 /* Tablet */
 
 /* Desktop */
+@media screen and (min-width: 768px) {
+  header .logo {
+    display: none;
+  }
+
+  nav > div {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    gap: 30px;
+  }
+
+  nav ul a {
+    margin-bottom: 20px;
+    color: #0a2759;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 30px;
+    font-style: normal;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+  }
```

Comment:
- [ ] [Optional] Kindly remove the `position: fixed` property as when the user scrolls down, the nav ie: `portfolio, about, and contact` overlap with the content of the page so I would suggest you remove the position: fixed

Reply:
@roshan-bajgain But I think even in the Figma it remains in the top of everything